sys-auth/google-oslogin/google-oslogin-20180611.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Components to support google cloud os login. This contains bits that belong in USR"
+HOMEPAGE="https://github.com/GoogleCloudPlatform/compute-image-packages/tree/master/google_compute_engine_oslogin"
+SRC_URI="https://github.com/GoogleCloudPlatform/compute-image-packages/archive/${PV}.tar.gz"
+
+LICENSE=""
+SLOT="0"
+KEYWORDS="amd64"
+IUSE=""
+
+# TODO: flesh this out
+DEPEND="
+	net-misc/curl[ssl]
+	dev-libs/json-c
+	virtual/pam
+"
+RDEPEND="${DEPEND}"
+
+src_unpack() {
+	S=${WORKDIR}/compute-image-packages-${PV}/google_compute_engine_oslogin
```

Comment:
This is usually set outside functions, and I think that makes it easier to find when the source uses an odd directory.  That could also allow you to omit the `src_unpack` definition if it's just calling `default`.